test/test.js

```diff
@@ -200,24 +200,6 @@ describe('moog', function() {
       });
     });
 
-    it('should create a subclass when the parent is an npm dependency of the subclass', function(done) {
```

Comment:
This test came as quite a surprise to me (: Obviously I did this to myself, however it has never been documented or supported as an Apostrophe or moog-require feature and it is inherently incompatible with the necessary fix made in this release.